src/app/content/components/SidebarControl.tsx

```diff
@@ -69,7 +69,7 @@ export const SidebarControl: React.SFC<InnerProps> = ({message, hideMobileText,
     {(msg: Element | string) => {
       const txt = assertString(msg, 'Aria label only supports strings');
       return <ToCButton aria-label={txt} {...props}>
-        <ListIcon/><ToCButtonText hideMobileText={!!hideMobileText}>Table of contents</ToCButtonText>
+        <ListIcon src={tocIcon}/><ToCButtonText hideMobileText={!!hideMobileText}>Table of contents</ToCButtonText>
```

Comment:
space missing

Reply:
accessibility tests are going to complain about the img with no aria or role or anything